Makefile

```diff
@@ -14,44 +18,45 @@ FFMPEG_OPTS ?= --prefix=$(FFMPEG_PREFIX) \
                --enable-parser=h264 \
                --enable-parser=hevc
 
-CGO_LDFLAGS := -L$(FFMPEG_PREFIX)/lib
+CGO_LDFLAGS := -L$(FFMPEG_BUILD)/lib
 ifeq ($(UNAME_S),Linux)
 	CGO_LDFLAGS := "$(CGO_LDFLAGS) -l:libjpeg.a"
 endif
 
-.PHONY: build-ffmpeg test lint updaterdk module clean
+.PHONY: build-ffmpeg lint update-rdk module clean clean-all
 
-bin/viamrtsp: build-ffmpeg *.go cmd/module/*.go
-	PKG_CONFIG_PATH=$(FFMPEG_PREFIX)/lib/pkgconfig \
+$(BIN_OUTPUT_PATH)/viamrtsp: build-ffmpeg *.go cmd/module/*.go
+	PKG_CONFIG_PATH=$(FFMPEG_BUILD)/lib/pkgconfig \
 		CGO_LDFLAGS=$(CGO_LDFLAGS) \
-		go build -o bin/viamrtsp cmd/module/cmd.go
-
-test:
```

Comment:
`make test` didn't work (I think b/c the `CGO_LDFLAGS` cause it to try to dynamically load the libav libs, but `go test` outside of make still works. 

As a result, I removed the target that didn't work